pkg/packagekit/updates.jsx

```diff
@@ -405,13 +405,13 @@ function UpdateHistory(props) {
         history = history.slice(0, props.limit);
 
     var paragraphs = history.map(pkgs => (
-        <div>
+        <React.Fragment>
             <p>{formatHeading(pkgs["_time"])}</p>
             <ul className='flow-list'>{formatPkgs(pkgs)}</ul>
-        </div>
+        </React.Fragment>
     ));
 
-    return <div>{paragraphs}</div>;
+    return <React.Fragment>{paragraphs}</React.Fragment>;
```

Comment:
Should this just be the following?

```jsx
return paragraphs;
```

Reply:
Yes, I think returning a list of elements is actually legitimate and works. I suppose I just didn't know about that yet when I wrote this. In the same vein, if Fragment does not work with react-lite, the other cases could actually use the same trick? I. e. return a (static) array?

Reply:
returning an array, no matter if static or generated, works fine. Just the `key` prop needs to be set